conda-env/proc.yml

```diff
@@ -2,7 +2,7 @@
 # This environment is for testing local source code changes to the `datasm` modules related to the data processing operations `extract`, `validate`, and `postprocess` before merging them to production (the `master` branch).
 name: datasm_proc
 channels:
-  - conda-forge/label/e3sm_dev
+  # - conda-forge/label/e3sm_dev
```

Comment:
I recommend commenting on why the `e3sm_dev` channel is used and why it is commented out now. Generally speaking, we shouldn't use `rc` packages in our production environments but it is up to your discretion.

Reply:
@tomvothecoder I'll be honest - I don't recall commenting that out (nor even what it means :( in any case).  I don't know what "rc packages" are, and how I would know if I was using any.  Apologies for my ignorance here.  Could this be related to accommodating the earlier "e3sm_to_cmip" test-releases?

I believe that "E2C" fixes to generating "CFmon" and "clcalipso" vars were made (and worked for v2) but are now failing with the v1_LE (large ensemble) datasets.  Could this be related?

Reply:
`rc` = release candidates

We wanted to use `nco` release candidates (e.g., `nco=5.11.0rc1` or something) which were uploaded on the `e3sm_dev` label of `conda-forge`. We temporarily added that channel and then commented it out because we needed the production/stable version of `nco`(e.g., `nco=5.11`) which is hosted on the `main` label of `conda-forge`.

Reply:
Thanks Tom!